cmd/virt-handler/BUILD.bazel

```diff
@@ -139,6 +139,17 @@ pkg_tar(
     package_dir = "/etc",
 )
 
+pkg_tar(
+    name = "nftables-tar",
+    srcs = [
+        ":ipv4-nat.nft",
+        ":ipv6-nat.nft",
+        ":nsswitch.conf",
```

Comment:
I don't think you wanted to pick up `nsswitch.conf` here.

Reply:
Done.